### PR TITLE
chore(kustomize): update cos-fleetshard kustomization images overlay dev

### PR DESCRIPTION
**Pull request by lburgazzoli**
sha: f58ffb6441105a290960eb6eb52a6b08feb21c4f, overlay: dev, tag: main-f58ffb6441105a290960eb6eb52a6b08feb21c4f

### CODE REVIEW

**Comment by lburgazzoli** [comment]
@rinaldodev should we merge this one ?

File: kustomize/overlays/dev/data-plane/kustomization.yaml [hunk 1]
```diff
@@ -2,12 +2,14 @@ resources:
...
-    digest: sha256:cb2130c0fb46a6e097b41a95296a3863fdf38eed418416430c60e02b9646327b
+    digest: sha256:75c5e89b7f6dc314f0c7d50c49786dbb06d42fcfb7b16dd5546546a0edd5dc49
   - name: quay.io/rhoas/cos-fleetshard-sync
-    digest: sha256:8034d30827747d43e657bb534f31de7bc2a4efaed2ce5a5564a4244c5daadc66
+    digest: sha256:cd4d6d694a96ef101cd5a6d8680a019c7228983c765d3721574191a1a55303e1
   - name: quay.io/rhoas/cos-camel-k
     digest: sha256:f42213014a5a5164abddb71984b24864e3b73044df2ab94ce76f5aec42b2864c
   - name: quay.io/strimzi/operator
     digest: sha256:8b010c5fb8244b742a37f264feefc116d75bb2abd2d6895375cbeea250727b39
+  - name: "null"
```
Comment (rinaldodev) [inline kustomize/overlays/dev/data-plane/kustomization.yaml:14]: > @rinaldodev should we merge this one ?

I don't think so

Comment (lburgazzoli) [reply kustomize/overlays/dev/data-plane/kustomization.yaml:14]: uh

Comment (heiko-braun) [reply kustomize/overlays/dev/data-plane/kustomization.yaml:14]: Haha, a good example demonstrating the value of PR reviews ;)

**Pull request closed without merging**